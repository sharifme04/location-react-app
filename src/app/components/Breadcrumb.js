import React from "react";

const Breadcrumb = () => {
  return (
    <div>
      <nav aria-label="breadcrumb">
        <ol className="breadcrumb">
          <li className="breadcrumb-item">Rundum mobil</li>
          <li className="breadcrumb-item">Preis- und Umweltvergleich</li>
        </ol>
      </nav>
    </div>
  );
};

export default Breadcrumb;
