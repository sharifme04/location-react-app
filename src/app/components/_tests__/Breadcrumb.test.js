import React from "react";
import renderer from "react-test-renderer";
import Breadcrumb from "../Breadcrumb";


test("snapshot Form renders with empty props", () => {
  const component = renderer.create(<Breadcrumb />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});