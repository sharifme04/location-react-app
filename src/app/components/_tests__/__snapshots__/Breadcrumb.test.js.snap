// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot Form renders with empty props 1`] = `
<div>
  <nav
    aria-label="breadcrumb"
  >
    <ol
      className="breadcrumb"
    >
      <li
        className="breadcrumb-item"
      >
        Rundum mobil
      </li>
      <li
        className="breadcrumb-item"
      >
        Preis- und Umweltvergleich
      </li>
    </ol>
  </nav>
</div>
`;
